lib/actions/adapterActions.js

```diff
@@ -1691,22 +1668,20 @@ export function openCustomAdapter(portPath) {
 export function initAdapter(device) {
     return dispatch => {
         if (device.serialport) {
-            const { serialNumber } = device;
             // Prefer to use the serialport 8 property or fall back to the serialport 7 property
-            const portPath =
-                device.serialport.path || device.serialport.comName;
+            const portPath = device.serialport.comName;
```

Comment:
This now does not match the comment above anymore. I think `comName` is supposed to be the deprecated property. Why are you not using `path` anymore?

Reply:
Because on Windows (which is the only one I've tested on), the `path` attribute returned by `nrf-device-lib-js` is something like:
![image](https://user-images.githubusercontent.com/72191781/134632258-f514a9c0-d22e-46c1-b040-d61c6a5b4162.png)

And the serialport module does not accept this type of `path` as input. But you are right about the comment.

Reply:
Ah, of course, the old code was for `node-serialport` but I guess this is not used here anymore. Do we still use `node-serialport` anywhere? I guess this means we should remove the use of `serialport.path` everywhere?